members/people.json

```diff
@@ -654714,10 +654714,12 @@
       "organization_id": "london-assembly", 
       "person_id": "uk.org.publicwhip/person/10169", 
       "post_id": "uk.org.publicwhip/cons/10839", 
-      "start_date": "2016-05-05"
+      "start_date": "2016-05-05", 
+      "start_reason": "regional_election"
     }, 
     {
       "end_date": "2016-05-05", 
+      "end_reason": "unknown", 
```

Comment:
This should also be regional_election (assuming this person stepped down/lost their seat, going by the date). Don't know if you can work that out from your data, or manually change after?